app/view_models/Amount.scala

```diff
@@ -39,6 +39,8 @@ case class Amount(amount: BigDecimal, currency: String) {
 
   def isZero = amount == BigDecimal(0)
```

Comment:
This comparison won't work, will need to use something like (pulled from tax-summaries)

```
trait DoubleUtils {

  implicit class Ops(d: Double) {

    def ===(that: Double): Boolean = (d - that).abs <= 1.0e-8 * d.abs
    def !==(that: Double): Boolean = !(d === that)
  }
}
```